nixos/doc/manual/development/running-nixos-tests-interactively.section.md

```diff
@@ -4,19 +4,19 @@ The test itself can be run interactively. This is particularly useful
 when developing or debugging a test:
 
 ```ShellSession
-$ nix-build nixos/tests/login.nix -A driverInteractive
+$ nix build -f . nixosTests.login.driverInteractive
 $ ./result/bin/nixos-test-driver --interactive
```

Comment:
It's a little surprising that this isn't the default for the interactive variant. Not necessarily in scope for this PR I guess.

Reply:
(I don't like the name "driverInteractive" in general, since the "non-interactive" regular driver also supports the `--interactive` flag...)